### PR TITLE
[Fix#9923] The alarm instance management interface is always in the loading state

### LINKED ISSUE #9923: [Bug] [Alarm Instance Manage] The alarm instance management interface is always in the loding state
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

When the alter component is not started, the alarm instance management interface is always in the loding state

### What you expected to happen

1. loading state can be turned off

### How to reproduce

1. first start api and master component 
2. start ui component 
3. click alarm Instance Manage

### Anything else

Kapture 2022-05-02 at 02 30 49

### Version

3.0.0-alpha

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### LINKED ISSUE #10207: [Bug] [Create Alarm Instance] When you enter the alarm instance management, the "Alarm Instance Component is Empty" error appears on the front end, and clicking "Create Alarm Instance" is always loading
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

image
image

### What you expected to happen

You can create an alarm instance

### How to reproduce

None.

### Anything else

_No response_

### Version

3.0.0-alpha

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
## Brief change log

close [#9923](https://github.com/apache/dolphinscheduler/issues/9923) also close #10207

## Verify this pull request

Manually verified the change by testing locally

### CODE REVIEW

**Comment by labbomb** [comment]
@SnowMoon-Dev When a request error has been caught and logged, this operation is useless

**Comment by SnowMoon-Dev** [comment]
> @SnowMoon-Dev When a request error has been caught and logged, this operation is useless
Yes, I know, I think our current handling of exceptions is a bit simple. In order to cope with subsequent changes, I have added log printing, and I will modify it.

**Comment by zhongjiajie** [comment]
@songjianet make sure this issue also close #10207

**Comment by zhongjiajie** [comment]
Hi @SnowMoon-Dev, if you could solve one issue in one single PR, you should not separate them into two or more, unless the issue contain more than one issue

**Comment by SnowMoon-Dev** [comment]
> Hi @SnowMoon-Dev, if you could solve one issue in one single PR, you should not separate them into two or more, unless the issue contain more than one issue



> Hi @SnowMoon-Dev, if you could solve one issue in one single PR, you should not separate them into two or more, unless the issue contain more than one issue

Okay, I know

**Comment by SnowMoon-Dev** [comment]
> @SnowMoon-Dev When a request error has been caught and logged, this operation is useless

I modified the relevant code, please help me review again

File: dolphinscheduler-ui/src/views/security/alarm-instance-manage/use-form.ts [hunk 1]
```diff
@@ -72,11 +72,15 @@ export function useForm() {
...
+    try {
+      const plugins = await queryUiPluginsByType({ pluginType: 'ALERT' })
+      state.uiPlugins = plugins.map((plugin: IPlugin) => ({
+        label: plugin.pluginName,
+        value: plugin.id
+      }))
+    } catch (e) {
+      state.pluginsLoading = false
+    }
     state.pluginsLoading = false
```
Comment (Amy0104) [inline dolphinscheduler-ui/src/views/security/alarm-instance-manage/use-form.ts]: Setting pluginsLoading to false should be within the try block.

Comment (SnowMoon-Dev) [reply dolphinscheduler-ui/src/views/security/alarm-instance-manage/use-form.ts]: ok，thanks

**Review by devosend — APPROVED** [review]
LGTM

**Pull request merged**